docs/observability.md

```diff
@@ -0,0 +1,118 @@
+|NgsaAppSummary<br>LodeRunnerSummary|struct| \[code, cosmos, mode, region, zone, quartil\]|Used to calculate the average duration across request types and filter by constituent values listed in **Range**. Quartile values are used to graph values that meet the 0.95 and 0.99 confidence intervals.<br>**Example query**: avg(NgsaAppSummary{namespace="\$namespace",code="OK",service="\$service",region=~"\$region",zone="azure",quantile=~"0.95|0.99"}) by (mode, quantile)|
+|NgsaAppSummary_countLodeRunner<br>Summary_count|struct|\[code, cosmos, mode, region, zone\]|Provides a running total of requests at a given time over a duration.<br>**Example query**: TBD|
+|NgsaAppSummary_sum<br>LodeRunnerSummary_sum|struct|\[code, cosmos, mode, region, zone\]|Sum of the summary values at a given instance.<br>**Example query**: TBD|
+|NgsaAppDuration_bucket<br>LodeRunnerSummary_bucket|struct|\[code, cosmos, mode, region, zone, le\]|Used to calculate the number of requests per intervale by a given mode (Query or Direct).<br>**Example query**: sum(rate(NgsaAppDuration_bucket{namespace="\$namespace",zone="azure",region=~"\$region",service="\$service"}\[1m\])) by (mode)|
+|NgsaAppDuration_count<br>LodeRunnerDuration_count|struct|\[code, cosmos, mode, region, zone\]|Used to calculate the requests per second over a given interval. Predicates may be used to filter by any of the field values.<br>**Example query**: sum(rate(NgsaAppDuration_count{namespace='\$namespace",zone="azure",region=~"\$region",service="\$service",mode != "Metrics"}\[1m\]))|
+|NgsaAppDuration_sum<br>LodeRunnerDuration_sum|struct|\[code, cosmos, mode, region, zone\]|Provides the sum of the duration values at a given instance.<br>**Example query**: TBD|
+
+Each of the above metrics has several field values that are used as part of the measurement.  The following table describes those sub-metric fields and their intended use:
+
+|**Field**|**Type**|**Range**|**Description**|
+| :-- | :-- | :-- | :-- |
+|mode|string|\[Query, Direct\]|Represent whether a request to CosmosDB is using a **direct** read by id or if it is using a **query** that must be parsed and its predicates used to filter.|
+|cosmos|bool|\[True, False\]|Indicates whether the app is running in in-memory mode or querying CosmosDB.This is not currently used in the WCNP deployment as the **service** field is used to filter for **ngsa-in-memory** or **ngsa-cosmos**|
+|region|string|Based on environment|This is set from the cluster environment and represents the geo region of the deployment.It is set in kitt.yml as:- --region={{$.kittExec.currentCluster.site}}|
+|zone|string|Based on environment|This is set from the cluster environment and represents the cloud used for the deployment.It is set in kitt.yml as:- --zone={{$.kittExec.currentCluster.provider}}|
+|code|string|\[Error, Retry, Warn, OK\]|This value represents the HTTP status disposition of the request:"Error" → 500+"Retry" → 429"Warn" → Any non-429 400 response"OK" → Any value < 400|
```

Comment:
Would be nice to have some newlines in the Description fields here for readability, especially for the code field

![image](https://user-images.githubusercontent.com/67930841/114211925-d9fd5f80-9926-11eb-8d38-0da4ff9f2ab7.png)

Reply:
Added <br> tags in certain spots and code block formatting for the flags to improve readability.